logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -33,7 +34,8 @@ actual class CallManager(
     private val calling: Calling,
     private val callRepository: CallRepository,
     private val userRepository: UserRepository,
-    private val clientRepository: ClientRepository
+    private val clientRepository: ClientRepository,
+    val messageSender: MessageSender
```

Comment:
By adding this new param we would need to pass it through `getCallManagerForClient` in `GlobalCallManager`